packages/injectable/src/dependency-injection-container/createContainer.registration.test.js

```diff
@@ -15,6 +16,24 @@ describe('createContainer.registration', () => {
     expect(actual).toBe('some-injected-instance');
   });
 
+  it('throws an error with the key if the value behind has no default export', () => {
+    const context = new Map();
+
+    context.set("./foobar.injectable.ts", {
+      notDefault: getInjectable({
+        id: 'irrelevant',
+        instantiate: () => 'some-injected-instance',
+      }),
+    });
+
+    expect(() => createContainer(() => Object.assign(
+      (key) => context.get(key),
+      {
+        keys: () => context.keys(),
+      }
+    ))).toThrowError(/\.\/foobar\.injectable\.ts/);
```

Comment:
Full literal error message is preferred for following reasons:
1. Documentation and readability.
2. Less logic in test is better.

Reply:
Okay... I chose not to because I think that specifying the exact literal would be too closely fitting the data (or not flexible enough to be able to change in the future without changing the test).

Reply:
In this case, having to change the test when the implementation changes is acceptable, if not desirable. Let's check out those behavioral tests elsewhere for a feel for tests that do not hinder refactorability, but instead enable it.